docs/book/src/forc/plugins/forc_client/index.md

```diff
@@ -74,6 +74,12 @@ Another alternative is the `--target` option, which provides useful aliases to a
 forc-deploy --target beta-3
 ```
 
+For delayed transactions, you can use the `--submit-only` flag. This flag allows you to submit the transaction without waiting for its finalization.
```

Comment:
We should make a separate header for this, since it isn't about "Interacting with the testnet" specifically. I think it's also useful to mention the multisig use case.

```suggestion

## Delayed transactions

For delayed transactions, you can use the `--submit-only` flag. This flag allows you to submit the transaction without waiting for its finalization.

One use case for this is multisig transactions, where a deployment transaction may stay in a pending state while waiting for all signatures.
```